hack/vendor.sh

```diff
@@ -38,6 +38,11 @@ clone git github.com/hashicorp/consul v0.5.2
 clone git github.com/docker/distribution 419bbc2da637d9b2a812be78ef8436df7caac70d
 clone git github.com/vbatts/tar-split v0.9.4
 
+clone git github.com/docker/notary 9926ead788e6a5657de62560a4fa25c0a771a2d8
+clone git github.com/endophage/gotuf 374908abc8af7e953a2813c5c2b3944ab625ca68
+clone git github.com/tent/canonical-json-go 96e4ba3a7613a1216cbd1badca4efe382adea337
```

Comment:
I said it already, but I really hope that in the future we could use our own implementation of this, with a properly documented set of difference with the standard golang package, and a way to track what version of the standard golang package it's based on (what's gonna happen when we move on to Go 1.5 here?).